lib/models/asyncapi.js

```diff
@@ -211,6 +223,69 @@ class AsyncAPIDocument extends Base {
   hasCircular() {
     return !!this._json[String(xParserCircle)];
   }
+
+  /**
+   * @callback TraverseSchemas
```

Comment:
is it correct? I see callback is called like this 👇 
`callback(schema, prop, SchemaIteratorCallbackType.NEW_SCHEMA)`

please also extend descriptions of each property as jsdoc doesn't show things like `NEW_SCHEMA/END_SCHEMA`

Reply:
The jsdocs have been updated to include descriptions and the enums 😄

Reply:
@jonaslagoni what about this

>is it correct? I see callback is called like this 👇
callback(schema, prop, SchemaIteratorCallbackType.NEW_SCHEMA)

don't you think that below table suggests you get the callback type as second argument, while in fact it is third? I think that order in jsdoc should change
<img width="823" alt="Screenshot 2020-12-07 at 13 16 23" src="https://user-images.githubusercontent.com/6995927/101349958-6f5fe880-388e-11eb-9c65-3ca813b9d837.png">

Reply:
🤔  good catch, did not even catch that in your comment 😄 That is changed.